component/Cards/card.css

```diff
@@ -0,0 +1,115 @@
+
+.parent-positioning img{
+    width:100%;
+    height:21rem;
+    object-fit:cover;
+    /* border-radius: 5px 5px 0px 0px; */
+}
+.vertical-cards{
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 3rem;
+}   
+.card-container{
+    display:flex;
+    flex-wrap:wrap;
+    max-width:27rem;
+    margin:0 auto;
+    margin-top:6rem;
+    background-color:var(--white-color);
+    box-shadow: 0px 3px 10px lightgray ;
+}
+.inside-container{
+    padding:8px 10px 3px 10px;
+    line-height: 2.5rem;
+}
+.card-content{
+    font-size: 1.6rem;
+    margin-top:1.5rem;
+}
+.card-button{
+    margin: 2rem 2rem 1rem 0;
+}
+.icon-size{
+    font-size: 2rem;
+}
+.parent-positioning{
+    position: relative;
+}
+.icon-overlay{
+    position: absolute;
+    top:1rem;
+    right:5px;
+    background-color: var(--white-color);
+    height:2.5rem;
+    width:2.7rem;
+    border-radius: 50%;
+    padding:3px 0 0 3.5px;
+}
+.icon-overlay:hover{
+    cursor: pointer;
+    color:var(--alert-error-color);
```

Comment:
suggestion: try putting variables in caps for better visibility in code

Reply:
will do that